frontend/src/modules/EditZing/Components/StudentCard.tsx

```diff
@@ -45,6 +83,32 @@ const StudentCard = ({
         width: '150px',
       }}
     >
+      <Snackbar
+        open={showNotesSaveSuccess}
+        autoHideDuration={3000}
+        onClose={() => setShowNotesSaveSuccess(false)}
+        message="Notes saved."
+      />
+      <Snackbar
+        open={showNotesSaveFailure}
+        autoHideDuration={3000}
+        onClose={() => setShowNotesSaveFailure(false)}
+        message="Notes failed to save."
+        ContentProps={{
+          style: {
+            background: '#ff3868',
```

Comment:
use `bgcolor: 'error.main'`. tbh this is optional, as the color isn't exactly the same, but that's part of the theme and it'll look cleaner than just a random hex val

Reply:
How is this possible? bgcolor isn't a prop or property right? And setting content props style background to error.main doesnt work either.

Reply:
Oh wait, I didn't look at the code carefully enough. Instead of style, you need to use sx.